azure-stack/hci/manage/arc-extension-management.md

```diff
@@ -43,12 +43,57 @@ You can install extensions from the **Capabilities** tab for your Azure Stack HC
 
 When you install an extension in the Azure portal, it's a cluster-aware operation. The extension is installed on all servers of the cluster. If you add more servers to your cluster, all the extensions installed on your cluster are automatically added to the new servers.
 
+
+
 ## Check the extension status
+# [Azure portal](#tab/azureportal)
```

Comment:
The heading following a H2 (##) should be a H3 (###). You can use a H4 (####) if the H3 seems a little too close to the size of the H2. Update Azure portal to **The Azure portal**.